common/scsi_debug

```diff
@@ -65,3 +65,30 @@ _exit_scsi_debug() {
 	unset SCSI_DEBUG_DEVICES
 	modprobe -r scsi_debug
 }
+
+# Args: [physical sector size, [logical sector size, [unaligned(0|1), [size in megs]]]]
+_get_scsi_debug_dev()
```

Comment:
There's already an `_init_scsi_debug` helper, please use that instead of adding this one.

Reply:
_get_scsi_debug_dev takes some options as input and returns a device back. Should _init_scsi_debug be modified to do something similar?

Reply:
You can pass arbitrary module parameters to `_init_scsi_debug` and it'll pass them along to modprobe. `_init_scsi_debug` sets `SCSI_DEBUG_DEVICES` instead of outputting a single block device. So for your test you can do
```
_init_scsi_debug lbpws=1 lbpws10=1
local dev="/dev/${SCSI_DEBUG_DEVICES[0]}"
```